window/glfw.go

```diff
@@ -184,6 +184,8 @@ const (
 	CursorDisabled = CursorMode(glfw.CursorDisabled)
 )
 
+// GlfwWindow describes one glfw window
+
 // GlfwWindow describes one glfw window
 type GlfwWindow struct {
```

Comment:
Why the extra lines?

Reply:
Lack of coffee... definitly not intention.